### ISSUE TITLE
Add ability to remove "auto add p tags" 

### ISSUE THREAD

**Issue by ComputerWarWizard** [andrew-worsfold/tailor#116]
First I want to say that I love this plugin and use it at work to help create wordpress sites quickly and still let our customer be able to edit the pages.  One thing that I constantly run into though is I will put in raw html to the editor text section but Tailor auto adds p tags still.  It would be nice if you could have a checkbox somewhere in the editor pane to disable that on an individual element level.

**Comment by andrew-worsfold** [comment]
The editor provided in content elements is just an instance of the default WordPress editor.  Is the behaviour you're seeing different from what you'd see if using the editor in the backend?

**Comment by ComputerWarWizard** [comment]
Yes it is different.  If I put in raw html in the text tab it will look how I expect it to.  With tailor when I do that it auto adds <p> tags where I have a new line
